rule/rule.go

```diff
@@ -721,8 +721,20 @@ type Rule struct {
 
 // NewRule creates a new, empty rule with the given kind and name.
 func NewRule(kind, name string) *Rule {
+	var call *bzl.CallExpr
 	kindIdent := &bzl.Ident{Name: kind}
-	call := &bzl.CallExpr{X: kindIdent}
+
+	parts := strings.Split(kind, ".")
+	if len(parts) > 1 {
+		// Create the same `CallExpr` like parser does
+		call = &bzl.CallExpr{X: &bzl.DotExpr{
+			Name: strings.Join(parts[1:], "."),
```

Comment:
I would have expected multiple `.`s to result in nested `DotExpr`s. Which existing part of Gazelle doesn't do this?

Reply:
Yeah, you're right. I should have checked it more carefully. I'll correct that.